src/controllers/dynakube/activegate/consts/consts.go

```diff
@@ -59,4 +61,26 @@ const (
 	GatewaySslMountPoint     = "/var/lib/dynatrace/gateway/ssl"
 	GatewayLogMountPoint     = "/var/log/dynatrace/gateway"
 	GatewayTmpMountPoint     = "/var/tmp/dynatrace/gateway"
+
+	DockerImageUser  int64 = 1001
+	DockerImageGroup int64 = 1001
+)
+
+var (
+	ContainerSecurityContext = corev1.SecurityContext{
```

Comment:
I would put that into a method of the kubemon modifier like `getSecurityContext()`

Reply:
done